src/dodal/devices/zocalo/zocalo_results.py

```diff
@@ -79,6 +92,7 @@ def __init__(
         self._prefix = prefix
         self._raw_results_received: Queue = Queue()
         self.transport: CommonTransport | None = None
+        self.use_cpu_and_gpu_zocalo = use_cpu_and_gpu_zocalo
```

Comment:
Nit: This device is already has `zocalo` in the name so having this variable with `zocalo` in is a bit redundant. It would also be good to have a docstring on it